Modules/Microsoft365DSC/DSCResources/MSFT_AADGroupsSettings/MSFT_AADGroupsSettings.psm1

```diff
@@ -68,7 +68,6 @@ function Get-TargetResource
         -InboundParameters $PSBoundParameters `
         -ProfileName 'beta'
 
-    $MaximumFunctionCount = 32000
```

Comment:
Shouldn't this change be in the changelog as well?

Reply:
I don't think we need to make a mention of this to be honest. This was fixed by the previous release this week and therefore no longer needed.